### PR TITLE
fix: dapp env vars (DOP-316)

### PR DESCRIPTION
**Pull request by witherblock**

### CODE REVIEW

File: apps/dapp/src/components/bonds/ModalBonds.tsx [hunk 1]
```diff
@@ -92,7 +93,7 @@ export const ModalBonds = ({ modalOpen, handleModal }: ModalBondsProps) => {
 
   useEffect(() => {
     async function getData() {
-      const response = await axios.get(`https://api.dopex.io/v2/price/dpx`);
+      const response = await axios.get(`${DOPEX_API_BASE_URL}/v2/price/dpx`);
```
Comment (garyunwin42) [inline apps/dapp/src/components/bonds/ModalBonds.tsx:96]: why do we need `DOPEX_API_BASE_URL` to be an env var? it's not a secret and we only have an env

Comment (witherblock) [reply apps/dapp/src/components/bonds/ModalBonds.tsx:96]: Its standard practice to move variables to the environment when they can change during testing and development purposes. Environment variables are not necessarily always used for secrets.

File: apps/dapp/src/constants/index.ts [hunk 2]
```diff
@@ -416,9 +414,7 @@ export const PAGE_TO_SUPPORTED_CHAIN_IDS: {
   '/straddles/DPX': { default: 42161, all: [42161] },
```
[the remark this answers is not in the document]
Comment (witherblock) [reply apps/dapp/src/constants/index.ts:414]: This is not the purpose of the PR.

[the remark this answers is not in the document]
Comment (garyunwin42) [reply apps/dapp/src/constants/index.ts:414]: why did you remove [these](https://github.com/dopex-io/elvarg/pull/429/files#diff-ed0451109505564c88af4f72bbadbbe28b49b3e363050c9372b8c60884dc68acL419-L421)?

File: apps/dapp/.env.example [hunk 3]
```diff
@@ -1,3 +1,5 @@
-NEXT_PUBLIC_INFURA_PROJECT_ID=
-NEXT_PUBLIC_ANKR_KEY=
 NEXT_PUBLIC_DEFAULT_CHAIN_ID=
+NEXT_PUBLIC_ANKR_KEY=
+NEXT_PUBLIC_INFURA_PROJECT_ID=
+NEXT_PUBLIC_GMX_STATS_API=
```
Comment (hibou-de-nuit) [inline apps/dapp/.env.example:4]: NEXT_PUBLIC_GMX_STATS_API_URL

**Review by hibou-de-nuit — CHANGES_REQUESTED** [review]
LGTM. update .env.local variable

File: apps/dapp/src/constants/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (witherblock) [reply apps/dapp/src/constants/index.ts:414]: Uh it was supposed to be a part of a separate PR, got intermingled

**Review by garyunwin42 — APPROVED** [review]

**Pull request merged**